src/AutoRotatingCarousel.js

```diff
@@ -150,9 +150,9 @@ export default class AutoRotatingCarousel extends Component {
   }
 
   render () {
-    const style = this.props.mobile ? mobileStyles : desktopStyles
+		const inlineStyles = this.props.inlineStyles ? Object.assign({}, desktopStyles, this.props.inlineStyles) : inlineStyles
```

Comment:
This doesn't allow overriding the inline styles when in mobile mode. Also, using object destructuring (`{ ...desktopStyles, ...this.props.inlineStyles }` would be preferable to me.